src/room/useGroupCall.ts

```diff
@@ -395,6 +403,43 @@ export function useGroupCall(groupCall: GroupCall): UseGroupCallReturnType {
     }
   }, [t]);
 
+  useEffect(() => {
+    const keyDownListener = (event) => {
+      // Check if keyboard shortcuts are enabled
+      const keyboardShortcuts = getSetting("keyboard-shortcuts", true);
+      if (!keyboardShortcuts) {
+        return;
+      }
+
+      if (event.key === "m") {
+        toggleMicrophoneMuted();
+      }
+      if (event.key === " ") {
+        setMicrophoneMuted(false);
+      }
+    };
+
+    const keyUpListener = (event) => {
+      // Check if keyboard shortcuts are enabled
+      const keyboardShortcuts = getSetting("keyboard-shortcuts", true);
+      if (!keyboardShortcuts) {
+        return;
+      }
+
+      if (event.key === " ") {
+        setMicrophoneMuted(true);
```

Comment:
Note that there are also gotchas here when the windows gets unfocused while the user has the button held which means the app never gets the keyup event. The PTT code should handle this so we may want to re-use the code from there, although it's probably a thing we could do later.

Reply:
So it looks to me like PTT auto-mutes if the window loses focus. I think the equivalent here would be to mute if the window loses focus while the space bar is down? I'm not sure which is more intuitive: that it mutes or stays unmuted in that case.

Happy to make the change, though I would need some guidance on how best to track a variable for "is the spacebar currently down?"

Reply:
sgtm - probably easiest to just add another state variable (`const { spacebarHeld, setSpacebarHeld } = useState(false);`) and then `setSpacebarHeld()` with the appropriate values in keydown/keyup and when the window loses focus.

Reply:
Done.